pages/admin.js

```diff
@@ -2,11 +2,13 @@ import React from "react";
 import Dashboard from "../components/admin_real/shared/Dashboard";
 import { withPageAuth } from "@supabase/auth-helpers-nextjs";
 
-export const getServerSideProps = withPageAuth({ redirectTo: "/signin" });
-
 // Example: https://github.com/vercel/nextjs-subscription-payments
 // Current Issue: https://github.com/supabase/auth-helpers/issues/281
 
-export default function Admin() {
+export default function Admin({ user }) {
   return <Dashboard />;
 }
+
+export const getServerSideProps = withPageAuth({
```

Comment:
why is the user getting redirected to /admin? From what I understand, redirectTo specifies the page that the user will be redirected to if the user is not authenticated. This doesn't make sense to me